beacon_node/network/src/service.rs

```diff
@@ -809,7 +810,7 @@ impl<T: BeaconChainTypes> NetworkService<T> {
             }
         } else {
             for column_subnet in DataColumnSubnetId::compute_custody_subnets::<T::EthSpec>(
-                self.network_globals.local_enr().node_id().raw().into(),
+                U256::from_le_slice(self.network_globals.local_enr().node_id().raw().as_slice()),
```

Comment:
this can panic if the value is larger than what can fit in u256. that should never happen here, but flagging it just in case. theres a few other places in this PR where we've done things similarly.